src/main/webapp/components/search-form-select/search-form-select.tsx

```diff
@@ -48,9 +72,16 @@ const Section = (props: SectionProps) => {
         <div
           style={{ display: 'flex', flexDirection: 'column', width: '100%' }}
         >
-          {searchForms.map(form => {
+          {searchForms.map((form, index) => {
             return (
-              <Button style={{ textAlign: 'left', width: '100%' }}>
+              <Button
+                key={form.id || index}
```

Comment:
in what case would the form not have an id?

Reply:
was a mistake